rfcs/0072-commonmark-docs.md

```diff
@@ -0,0 +1,334 @@
+---
```

Comment:
For some reason `0072-commonmark-docs.md` is part of this RFC, could you remove it?

Reply:
Squashed the edits until now. Seems to have helped.